src/index.js

```diff
@@ -14,4 +13,4 @@ root.render(
 // If you want to start measuring performance in your app, pass a function
```

Comment:
I dont think the comment s on lines 13 to 15 are necessary, what do you think?

Reply:
Yes its not necessary